.github/workflows/go.yml

```diff
@@ -208,7 +214,13 @@ jobs:
               working-directory: go
               run: |
                   make install-tools-go${{ matrix.go }} build
-                  make -k unit-test integ-test example-test
+                  make -k unit-test integ-test
+
+            - name: Run Example Tests
+              working-directory: go
+              if: ${{ matrix.engine.version != '6.2' }}
+              run: |
+                  make -k example-test
```

Comment:
Do we need to repeat examples here too? I think it is enough to run these tests once per matrix.

Reply:
I am just duplicating the existing structure. I've removed this additional set of tests in the containers